content/en/blog/extend-falco-inputs-with-a-plugin-register.md

```diff
@@ -0,0 +1,112 @@
+
+The table in the [README](https://github.com/falcosecurity/plugins#registered-plugins) is auto generated by aforementioned registry:
+
+| ID  | Name                                                                                      | Event Source     | Description                                                                                                                                                                                                                   | Info                                                                                |
+| --- | ----------------------------------------------------------------------------------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- |
+| 2   | [cloudtrail](https://github.com/~~falcosecurity~~/plugins/tree/master/plugins/cloudtrail) | `aws_cloudtrail` | Reads Cloudtrail JSON logs from files/S3 and injects as events                                                                                                                                                                | Authors: [The Falco Authors](https://falco.org/community) <br/> License: Apache-2.0 |
+| 3   | [dummy](https://github.com/falcosecurity/plugins/tree/master/plugins/dummy)               | `dummy`          | Reference plugin used to document interface                                                                                                                                                                                   | Authors: [The Falco Authors](https://falco.org/community) <br/> License: Apache-2.0 |
+| 4   | [dummy_c](https://github.com/falcosecurity/plugins/tree/master/plugins/dummy_c)           | `dummy_c`        | Like Dummy, but written in C++                                                                                                                                                                                                | Authors: [The Falco Authors](https://falco.org/community) <br/> License: Apache-2.0 |
+| 999 | test                                                                                      | `test`           | This ID is reserved for source plugin development. Any plugin author can use this ID, but authors can expect events from other developers with this ID. After development is complete, the author should request an actual ID | Authors: N/A <br/> License: N/A                                                     |
+
+# Details of your plugin
+
+In this section, we'll describe the key elements for describing your plugin.
```

Comment:
Can you please elaborate more what "describing" means in this context? Is this some kind of `YAML` file that we need to fill out the fields?